test/server/types.ts

```diff
@@ -0,0 +1,62 @@
+import { expect, test } from 'vitest'
+import { app } from './utils'
+
+test('type list filtering', async () => {
```

Comment:
also same as for other list tests, we should create some first to check that they are returned and that limit works (not sure we have >5)